_plugins/code_parser.rb

```diff
@@ -0,0 +1,20 @@
+# Description : This plugin checks code blocks created using ``` for ASCCI characters that can cause problem pasted to terminal.
```

Comment:
UTF-8* Not ASCII.

ASCII is usable by kubectl / vast majority of cross-platform tooling. The universal UTF-8 conversion is well underway but it's been in progress for 20 years 😄

Reply:
I started this plugin by converting everything to ASCII and checking for its Decimal representation!
I have changed everything but seems like that miss spelled ASCII was extremely lucky :D